### PR TITLE
Add designator when events are from the api

### PR DESCRIPTION
**Pull request by NickM-27**
Will make it possible for #6989 to only be shown for custom events

### CODE REVIEW

**Comment by skrashevich** [comment]
events from audio detector will be marked as "type: api"....

File: frigate/events/maintainer.py [hunk 1]
```diff
@@ -229,10 +230,10 @@ def handle_external_detection(self, type: str, event_data: Event) -> None:
                 Event.has_clip: event_data["has_clip"],
                 Event.has_snapshot: event_data["has_snapshot"],
                 Event.zones: [],
-                Event.data: { "type": "api" },
+                Event.data: {"type": event_data["type"]},
             }
             Event.insert(event).execute()
-        elif type == "end":
+        elif event_type == "end":
```
Comment (skrashevich) [inline frigate/events/maintainer.py:236]: you're broken all double-take installations now. at minimum

Comment (NickM-27) [reply frigate/events/maintainer.py:236]: in what way would this affect double-take at all?

Comment (NickM-27) [reply frigate/events/maintainer.py:236]: The code you highlighted is just changing the name of a local variable

Comment (skrashevich) [reply frigate/events/maintainer.py:236]: In no way. My mistake :(

**Comment by skrashevich** [comment]
it's a final version? Can I made corresponding changes to #6989 ?

**Comment by NickM-27** [comment]
yes, pending any review comments / requested changes

**Review by blakeblackshear — APPROVED** [review]

**Pull request merged**